<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoregSettingsViewWidget</class>
 <widget class="QWidget" name="CoregSettingsViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HPI Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_qTabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m_qTab_LoadPrepare">
      <attribute name="title">
       <string>Load and Prepare</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_MriSubject">
         <property name="title">
          <string> MRI Subject</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="m_qPushButton_BemFileDialog">
            <property name="text">
             <string>Load BEM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_qLineEdit_BemFileName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Fiducials">
         <property name="title">
          <string>MRI Fiducials</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="m_qPushButton_FidFileDialog">
            <property name="text">
             <string>Load Fiducials</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_qLineEdit_FidFileName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_qWidget_PickFiducials" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="m_qPushButton_PickLPA">
               <property name="text">
                <string>LPA</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_qPushButton_PickNas">
               <property name="text">
                <string>Nasion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_qPushButton_PickRPA">
               <property name="text">
                <string>RPA</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_qLineEdit_FidStoreFileName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>*-fiducials.fif</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_qPushButton_FidStoreFileDialog">
            <property name="text">
             <string>Store Fiducials</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Digitizer">
         <property name="title">
          <string>Digitizers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="m_qPushButton_DigFileDialog">
            <property name="text">
             <string>Load Digitizer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_qLineEdit_DigFileName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_qWidget_Discard" native="true">
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="m_qLabel_Text">
               <property name="text">
                <string>Points Omitted:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="m_qLabel_NOmitted">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="m_qSpinBox_MaxDist">
               <property name="suffix">
                <string> mm</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Max. distance to surface: </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_qTab_Fit">
      <attribute name="title">
       <string>Fit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_FittingOptions">
         <property name="title">
          <string>Fitting Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="m_qCheckBox_AutoScale">
            <property name="text">
             <string>Apply automatic scaling</string>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_qWidget_IcpIterations" native="true">
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>ICP iterations (max):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="m_qSpinBox_MaxIter"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Convergence Limit:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="m_qDoubleSpinBox_Converge">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="singleStep">
                <double>0.000100000000000</double>
               </property>
               <property name="value">
                <double>0.000100000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_qWidget_Weigths" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="m_qDoubleSpinBox_WeightNas">
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="m_qDoubleSpinBox_WeightLpa">
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="m_qDoubleSpinBox_WeightRpa">
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>LPA: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Nasion: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>RPA: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_qPushButton_FitFiducials">
            <property name="text">
             <string>Fit Fiducials</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_qPushButton_FitICP">
            <property name="text">
             <string>Fit ICP</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_StoreTrans">
         <property name="title">
          <string>Store Transformation</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="m_qLineEdit_TransFileStore">
            <property name="text">
             <string>*-trans.fif</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_qPushButton_TransFileStoreDialaog">
            <property name="text">
             <string>Store Transformation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_qTab_Process">
      <attribute name="title">
       <string>Adjustment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Scaling">
         <property name="title">
          <string>Scaling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="m_qWidgetScling" native="true">
            <layout class="QFormLayout" name="formLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="m_qSpinBox_X">
               <property name="suffix">
                <string> %</string>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="m_qSpinBox_Y">
               <property name="suffix">
                <string> %</string>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="m_qSpinBox_Z">
               <property name="suffix">
                <string> %</string>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Scaling Mode:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="m_qComboBox_ScalingMode">
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Uniform</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3-Axis</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_qPushButton_ApplyScaling">
            <property name="text">
             <string>Apply Scaling</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Results">
         <property name="title">
          <string>Results and Adjustment</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Translation [mm]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Rotation  [Â°]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>X: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_qLineEdit_TransX"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="m_qLineEdit_RotX"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Y: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="m_qLineEdit_TransY"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="m_qLineEdit_RotY"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Z: </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="m_qLineEdit_TransZ"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="m_qLineEdit_RotZ"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>456</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
